Formula/caddy@1.rb

```diff
@@ -0,0 +1,66 @@
+class CaddyAT1 < Formula
```

Comment:
Please indicate how this would fill our criteria for versioned formulas (https://docs.brew.sh/Versions) and why we might want to ship an older version?

Reply:
It fulfills all standards listed in the docs (except `keg_only :versioned_formula`, which I forgot to add and will do with the other fixes).

The reason why this should be kept for now is that Caddy 2 is a complete rewrite with many changes that will mean that users of Caddy 1 may want to keep using Caddy 1 until they have migrated to Caddy 2.

Reply:
Is there a planned end of life for caddy 1? Or will it be supported indefinitely?

Reply:
Caddy 1 is not actively being developed anymore, but I suggest to keep it the formula for a few months until everyone had the chance to switch. There is no fixed EOL date.

Reply:
If it's not actively developed it doesn't meet all criteria in the link. You could maintain an unsupported version in a tap if needed.

Reply:
I don't see that in the list. It only says that there should be a release branch and security support (which both still apply). It does not say that there needs to be active development.

But in the end the `caddy@1` formula is not something I care about. I wanted to keep it around for other people who will likely still need it, but if you can't allow that, it's fine for me too.

Reply:
Yeah, I think we confused each other with terminoligy, we can review a PR for caddy@1 if someone actually needs it.

Reply:
That sounds reasonable. 👍